test/UnboxPragma.agda

```diff
@@ -10,6 +10,13 @@ open ∃ public
 
 {-# COMPILE AGDA2HS ∃ unboxed #-}
 
+record Σ0 (A : Set) (P : @0 A → Set) : Set where
+  field
+    @0 el : A
+    pf : P el
+
+{-# COMPILE AGDA2HS Σ0 unboxed #-}
```

Comment:
The associated golden file is missing.

Reply:
I expanded an existing test, and the new record shouldn't produce any output so the golden file should stay the same.